recipes/libid3tag/all/conanfile.py

```diff
@@ -0,0 +1,107 @@
+from conans import ConanFile, tools, AutoToolsBuildEnvironment, MSBuild
+from conans.errors import ConanInvalidConfiguration
+import os
+
+
+class LibId3TagConan(ConanFile):
+    name = "libid3tag"
+    description = "ID3 tag manipulation library."
+    topics = ("conan", "mad", "id3", "MPEG", "audio", "decoder")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://www.underbit.com/products/mad/"
+    license = "GPL-2.0-or-later"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {"shared": [True, False], "fPIC": [True, False]}
+    default_options = {"shared": False, "fPIC": True}
+    requires = "zlib/1.2.11"
```

Comment:
Without a Conan generator, this library is not going to consume the Conan requirement. This job must be linking with the system one.